doc/api/fs.md

```diff
@@ -1280,8 +1280,8 @@ changes:
 -->
 
 * `file` {string|Buffer|URL|FileHandle} filename or `FileHandle`
-* `data` {string|Buffer|TypedArray|DataView|Object|AsyncIterable|Iterable
-  |Stream}
+* `data` {string|Buffer|TypedArray|DataView|AsyncIterable|Iterable|Stream|
+Object}*
```

Comment:
Re-ordered so `Object` is last, and add `*`

Reply:
The `*` doesn't refer to anything until 2500 lines below?

Reply:
Sorry, I didn't mean a literal asterisk. I would just add the language used in https://github.com/nodejs/node/pull/34993 only to the parts where only the metadata and data types were updated and whose description doesn't consist of a documentation "redirect".

The only API I can see that seems to be missing the wording and doesn't have a documentation "redirect" is `filehandle.write(buffer[, offset[, length[, position]]])`.

Reply:
Also I think the "(not inherited)" part is redundant, as that's implied with an "own property".

Reply:
> Also I think the "(not inherited)" part is redundant, as that's implied with an "own property".

Yes, intentionally redundant though: I added it to specifically call out / emphasis the `toString()` cannot come from where the user would reasonably expect. I feel like with gotchas (and this is a very bizarre gotcha), they should be abundantly clear.

Reply:
Then please make sure to update all object `toString()` references in this document for consistency.